Packs/FeedGithub/Integrations/FeedGithub/FeedGitHub.py

```diff
@@ -0,0 +1,514 @@
+            "type": type_,
+            "service": "github",
+            "fields": {},
+            "rawJSON": raw_data,
+        }
+
+        if feed_tags:
+            indicator_obj["fields"]["tags"] = feed_tags
+
+        if tlp_color:
+            indicator_obj["fields"]["trafficlightprotocol"] = tlp_color
+
+        indicators.append(indicator_obj)
+    demisto.debug(f"After fetch command last run: {last_commit_info}")
+    if last_commit_info:
+        demisto.setLastRun(last_commit_info)
+
+    return indicators
+
+
+def get_indicators(client: Client, last_commit_fetch=None):
+    indicators = []
+
+    relevant_files, last_commit_info = get_commits_files(client, last_commit_fetch)
+    feed_type = str(demisto.params().get("feedType"))
+    files_tokens = get_files_token(client, relevant_files)
+    try:
+        if feed_type == "YARA":
+            indicators = get_yara_indicator(files_tokens)
+            demisto.debug(f"YARA indicators : {indicators}")
+
+        elif feed_type == "STIX":
+            stix_client = STIX2XSOARParser({})
+            # files_tokens = eval((demisto.params()).get("data"))
+            generator_stix_files = create_generator(files_tokens)
+            indicators = stix_client.load_stix_objects_from_envelope(generator_stix_files)
+
+        elif feed_type == "IOCs":
+            for file in files_tokens:
+                indicators += extract_text_indicators(file)
+            demisto.debug(f"IOCs` indicators : {indicators}")
+
+    except Exception as err:
+        demisto.debug(str(err))
+        raise ValueError(f"Could not parse returned data as indicator. \n\nError massage: {err}")
+    return indicators, last_commit_info
+
+
+def get_indicators_command(client: Client) -> CommandResults:
+    """Wrapper for retrieving indicators from the feed to the war-room.
+    Args:
+        client: Client object with request
+        params: demisto.params()
+        args: demisto.args()
+    Returns:
+        Outputs.
+    """
+    indicators = []
+    try:
+        indicators, _ = get_indicators(client=client)
+
+        demisto.debug(f"indicators: {indicators}")
+        return CommandResults(
+            outputs_prefix=CONTEXT_PREFIX + ".Indicators",
+            outputs_key_field="githubfeed",
+            raw_response=indicators,
+            outputs=indicators,
+        )
+
+    except Exception as err:
+        demisto.debug(str(err))
+        raise ValueError(f"Could not parse returned data as indicator. \n\nError massage: {err}")
+
+
+def fetch_indicators_command(client: Client, params: Dict[str, str]) -> List[Dict]:
+    """Wrapper for fetching indicators from the feed to the Indicators tab.
+    Args:
+        client: Client object with request
+        params: demisto.params()
+    Returns:
+        Indicators.
+    """
+    feed_tags = argToList(params.get("feedTags", ""))
+    tlp_color = params.get("tlp_color")
+    last_commit_fetch = demisto.getLastRun().get("last_commit")
```

Comment:
```suggestion
    last_commit_fetch = demisto.getLastRun().get("last_commit") or get_commit_after_time(params.firstfetchtime)
```
the proper place to do this is here